apps/condo/domains/common/components/Table/Filters.tsx

```diff
@@ -121,31 +121,36 @@ export const getGQLSelectFilterDropdown = (
     mode?: ComponentProps<typeof GraphQlSearchInput>['mode'],
     containerStyles?: CSSProperties
 ) => {
-    return ({ setSelectedKeys, selectedKeys, confirm, clearFilters }) => {
+    return ({ setSelectedKeys, selectedKeys, confirm, clearFilters, visible }) => {
         const handleClear = useCallback(() => {
             isFunction(clearFilters) && clearFilters()
             setSelectedKeys('')
             confirm({ closeDropdown: true })
-        }, [clearFilters])
+        }, [clearFilters, confirm, setSelectedKeys])
+
+        const handleChange = useCallback((e) => {
+            setSelectedKeys(e)
+            confirm({ closeDropdown: false })
+        }, [confirm, setSelectedKeys])
+
         return (
-            <SelectFilterContainer
-                clearFilters={handleClear}
-                showClearButton={selectedKeys && selectedKeys.length > 0}
-                style={containerStyles}
-            >
-                <GraphQlSearchInput
-                    style={GRAPHQL_SEARCH_INPUT_STYLE}
-                    search={search}
-                    showArrow
-                    mode={mode}
-                    value={selectedKeys}
-                    onChange={(e) => {
-                        setSelectedKeys(e)
-                        confirm({ closeDropdown: false })
-                    }}
-                    {...props}
-                />
-            </SelectFilterContainer>
+            visible && (
```

Comment:
shouldn't be here some kind of placeholder ?

Reply:
No, you just need to hide/show the element